### PR TITLE
Manual Vault token refresh (Increase token timeouts). PT-163087449

### PR DESCRIPTION
**Pull request by valkov-ivaylo**
Add script to manually refresh Vault tokens (and leases). 
Updated README file regarding refreshing Vault secrets.

### CODE REVIEW

File: refresh-secrets.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+vault token revoke -self
+unset VAULT_TOKEN
+source /infrastructure/import-secrets.sh
```
Comment (dincho) [inline refresh-secrets.sh]: This should be a relative path. I often use the container with `/src` mounted.

Comment (valkov-ivaylo) [reply refresh-secrets.sh]: From my understanding `/infrastructure` should always be present. Relative to what? Current working directory could be any. The source will not find the file then. What am I missing?

Comment (dincho) [reply refresh-secrets.sh]: During the development I use `/src` mount for example, if this is kept as is, I need to rebuild the container each time I change the `import-secrets.sh` to test the `refresh-secrets.sh`

Comment (valkov-ivaylo) [reply refresh-secrets.sh]: Fixed.

**Pull request merged**